src/components/Tabs/TabItem.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, ReactElement, useState } from "react";
+import { Badge, BadgeProps } from "@components/Badge";
+import { motion } from "framer-motion";
+import { merge } from "@utilities/merge";
+
+export type TabITemProps = {
+    decorator?: ReactElement;
+    badge?: BadgeProps;
+    label: string;
+    onClick: () => void;
+    active: boolean;
+    disabled: boolean;
+};
+
+export const TabItem: FC<TabITemProps> = (props: TabITemProps) => {
```

Comment:
```suggestion
export const TabItem: FC<TabITemProps> = ({ onClick, label, active, decorator, badge, disabled = false }: TabItemProps) => {
```

So you can have easy default values